M2/Macaulay2/packages/Varieties.m2

```diff
@@ -59,6 +94,9 @@ runLengthEncoding := x -> if #x === 0 then x else (
      p := join({0}, select(1 .. #x - 1, i -> x#i =!= x#(i-1)), {#x});
      apply(#p-1, i -> (p#(i+1)-p#i, x#(p#i))))
 
+-- prints the message only if bool is false and debugLevel > 0
+condition = (bool, msg) -> bool or ( if debugLevel > 0 then printerr msg; false )
+
```

Comment:
I would prefer to see a more uniform naming convention used for the various unexported error checking subroutines.  For example, a consistent prefix like ```check``` might work well.  In this particular, case it could be ```checkCondition```.

Reply:
I agree that a uniform name is better. In this particular case I'd rather the name of this function be short and hopefully soon it should be moved to Core for use by other packages, so I'll change it to `assert'` (the point being that `assert` throws an error but `assert'` prints a warning if `debugLevel` is high)